src/main/java/seedu/address/commons/core/index/Index.java

```diff
@@ -8,7 +8,7 @@
  * base the other component is using for its index. However, after receiving the {@code Index}, that component can
  * convert it back to an int if the index will not be passed to a different component again.
  */
-public class Index {
+public class Index implements Comparable<Index> {
```

Comment:
Looks good, but can change to comparator class.

Reply:
Ok Noted thanks, will do.